src/registrar/models/domain.py

```diff
@@ -910,10 +910,15 @@ def renew(self):
         raise NotImplementedError()
 
     def get_security_email(self):
-        logger.info("get_security_email-> getting the contact ")
-        secContact = self.security_contact
-        if secContact is not None:
-            return secContact.email
+        logger.info("get_security_email-> getting the contact")
+
+        security = PublicContact.ContactTypeChoices.SECURITY
+        security_contact = self.generic_contact_getter(security)
+
+        # If we get a valid value for security_contact, pull its email
+        # Otherwise, just return nothing
+        if security_contact is not None and isinstance(security_contact, PublicContact):
```

Comment:
**comment/non blocking** interesting why was the isinstance needed?

Reply:
This PR originally had some changes to this particular function. As I was making those, I found that it was possible for this function to pass in the wrong type here. I don't think this is a problem we could encounter outside that though, its just a nice to have check